### PR TITLE
Add analytics for features shown

### PR DESCRIPTION
**Pull request by RByers**
Diff on top of #343.  How do I submit a PR on top of another PR?

### CODE REVIEW

**Comment by RByers** [comment]
Note that this is following the [GA user timing docs](https://developers.google.com/analytics/devguides/collection/analyticsjs/user-timings).

File: templates/features.html [hunk 1]
```diff
@@ -98,6 +99,12 @@
   $('#num-features').textContent = e.detail.count;
 });
 
+featureList.addEventListener('unveiled', function() {
+  console.timeStamp && console.timeStamp('features unveiled');
+  console.timeEnd && console.timeEnd('features unveil');
+  unveilStart && ga('send', 'timing', 'features', 'unveil', performance.now() - unveilStart);
```
Comment (ebidel) [inline templates/features.html]: we may want to protect against ios6. IIRC, `window.performance` is defined but `'window.performance.now` isn't :( Believe they shut it off.

Comment (RByers) [inline templates/features.html]: Ugh, good point - thanks.

Comment (RByers) [inline templates/features.html]: Done

**Comment by ebidel** [comment]
LGTM

**Pull request merged**